__tests__/app.test.js

```diff
@@ -112,8 +112,45 @@ describe("GET /api/articles", () => {
       .get("/api/banana")
       .expect(404)
       .then(({ body }) => {
-        console.log(body);
         expect(body.msg).toBe("Incorrect endpoint");
       });
   });
 });
+
+describe("GET /api/articles/articles_id/comments", () => {
+  test("200: should respond with an array of aall comment from article with a given article_id", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const allComments = body.comments;
+        allComments.forEach((eachComment) => {
+          expect(eachComment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
The comments for a specific article have been requested so we should assert the correct id on each comments